File: TSPractice/StringProblems/p12.ts
//Frequency of characters Appearance

export let s1 = "Hello Midhun, Welcome to Kerala";
export let char1 = "a";
function getFrequency(char2: String) : number {
    let frequency = 0;
    for(let charecters of s1.split("")){
        if(charecters == char2){
            frequency++;
        }
    }
    return frequency;
}
console.log(getFrequency(char1));

File: Java/src/StringProblems/p12.java
//Frequency of characters Appearance

package StringProblems;

public class p12 {

    public static int getFrequenct(String str, char charToFind) {
        int frequency = 0;
        for(char charecter: str.toCharArray()){
            if(charecter == charToFind){
                frequency++;
            }
        }
        return frequency;
    }
    public static void main(String[] args) {
        String s1 = "Hello Midhun, whelcome to Kerala";
        char ch = 'e';
        System.out.println(getFrequenct(s1, ch));
        }
}